.pre-commit-config.yaml

```diff
@@ -0,0 +1,67 @@
+# All available hooks: https://pre-commit.com/hooks.html
+# R specific hooks: https://github.com/lorenzwalthert/precommit
+repos:
+  - repo: https://github.com/lorenzwalthert/precommit
+    rev: v0.1.3.9137
+    hooks:
+      - id: style-files
+        args: [--style_pkg=styler, --style_fun=tidyverse_style]
+      - id: roxygenize
+        additional_dependencies:
+          - plumber
+          - shiny
+      # codemeta must be above use-tidy-description when both are used
+      # -   id: codemeta-description-updated
+      - id: use-tidy-description
+      - id: spell-check
+        exclude: >
+          (?x)^(
+          data/.*|
+          (.*/|)\.Rprofile|
+          (.*/|)\.Renviron|
+          (.*/|)\.gitignore|
+          (.*/|)NAMESPACE|
+          (.*/|)WORDLIST|
+          (.*/|)\.travis.yml|
+          (.*/|)appveyor.yml|
```

Comment:
We don't really use these, so we can remove them. Also, let's exclude the `.github` directory.